docs/conf.py

```diff
@@ -48,16 +48,16 @@
 
 # General information about the project.
 project = u'libCellML'
-copyright = u'2014, David Nickerson'
+copyright = u'2014-2015, University of Auckland'
```

Comment:
Just noticed that there are a few other places where we might want to remove David's name. Lines 204-207:

```
latex_documents = [
  ('index', 'libCellML.tex', u'libCellML Documentation',
   u'David Nickerson', 'manual'),
]
```

Lines 234-237:

```
man_pages = [
    ('index', 'libcellml', u'libCellML Documentation',
     [u'David Nickerson'], 1)
]
```

And lines 248-252:

```
texinfo_documents = [
  ('index', 'libCellML', u'libCellML Documentation',
   u'David Nickerson', 'libCellML', 'One line description of project.',
   'Miscellaneous'),
]
```